incubator/hnc/internal/forest/namespace.go

```diff
@@ -88,6 +92,14 @@ func (ns *Namespace) UnsetExists() bool {
 	return changed
 }
 
+func (ns *Namespace) GetLabels() map[string]string {
+	return ns.labels
+}
+
+func (ns *Namespace) SetLabels(labels map[string]string) {
+	ns.labels = labels
+}
+
```

Comment:
If we never do anything smart with these fields, maybe we should just make `ns.Labels` public?

Reply:
Or should we make a deep copy of the labels that get passed in so they can't be modified later? (`ns.labels = labels` is a shallow copy)

Reply:
done. Made deep copies for `ns.labels`